### PR TITLE
Sd kv0.1.0

### PR DESCRIPTION
**Pull request by aditim121**

### CODE REVIEW

File: cosmos1pkkayn066msg6kn33wnl5srhdt3tnu2vzasz9c.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"hashpwd":"b40d03386fb8fd7dc6ca9c6f8d680dfc58d60fd6ea72c63f573cc5bfbb09695746a849c91517c235cdfeaca4e734208f184bc08aae06797748988e95928f80b1","iv":"4c80a336adc039386ee27ded90395e46","salt":"7576f5ee41d120a2e22a34f856ec4e5b9ed9781efa5b22fb37f49e1011b6de43","crypted":"4824f8f09a8fe5c268c799ee514dff1b360a2df237e3e443c89f6ffae7e23804cc0357d5c938b3ea13d2ffff904da355cac888f83798511f11400d1ca38c042e2b290b990f12338c95fe6b2ddb449f8c27d203b093248feb5cd66420f0299ee16300f766131dc4319f6110b5c0835a88eeb13993f2d3cacb1fc39325cb2354f35ee3086faada4e7ce86cede8fa7d5606fa33eb7496112c763e2066090c5cad57"}
```
Comment (deepanshutr) [inline cosmos1pkkayn066msg6kn33wnl5srhdt3tnu2vzasz9c.json:1]: exclude file

File: cosmos160phc9aajqmllhn442tyehkrwpd8sytr6q9uwc.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"hashpwd":"b40d03386fb8fd7dc6ca9c6f8d680dfc58d60fd6ea72c63f573cc5bfbb09695746a849c91517c235cdfeaca4e734208f184bc08aae06797748988e95928f80b1","iv":"d4e08e7bbab8ae590d96001b33c0e059","salt":"8dd7e9c19399e2ff328f7f5c8ffa6f6b83312ca84daa3240292a86699b0f7ad3","crypted":"7c544a71f2fcaa490ce5663b363c1eaeaa901a0fae3a8f1db72718f84c2f04e379c99f06ac010af258eb8f34d1199cd63cd1a1777c022e37f0df9b6aa7352e917f68be85ab6b7b69edfda58100cb314971dcda08bb8957fcafd0d30b9bfe52a46e3ae1a02b78c30fc35782ff19e3f81d7f161e634b39fc6bb293a9248f859474c74b8d9c75b373ed355c2427a99e1c5da3cec55bdc0835cff12ade10976c86b1"}
```
Comment (deepanshutr) [inline cosmos160phc9aajqmllhn442tyehkrwpd8sytr6q9uwc.json:1]: exclude file

File: transaction/accounts/create.js [hunk 3]
```diff
@@ -21,38 +21,38 @@ function create(mnemonic, name) {
...
 
-                var options2 = {
+                let options2 = {
                     'method': 'POST',
                     'url': config.lcdURL + config.broadcastTx,
                     'headers': {
                         'Content-Type': 'application/json'
                     },
-                    body: JSON.stringify({"tx":{"msg":[{"type":"cosmos-sdk/MsgSend","value":{"from_address":config.testAccountAddress,"to_address":x.address,"amount":[{"denom":"stake","amount":"1000"}]}}],"fee":{"amount":[],"gas":"200000"},"signatures":[{"pub_key":{"type":typekey,"value":value},"signature":signature}],"memo":""},"mode":"sync"})
+                    body: JSON.stringify({"tx":{"msg":[{"type":"cosmos-sdk/MsgSend","value":{"from_address":address,"to_address":x.address,"amount":[{"denom":"stake","amount":"1000"}]}}],"fee":{"amount":[],"gas":"200000"},"signatures":[{"pub_key":{"type":typekey,"value":value},"signature":signature}],"memo":""},"mode":"sync"})
```
Comment (deepanshutr) [inline transaction/accounts/create.js:50]: remove hardcoding

File: transaction/bank/sendCoin.js [hunk 4]
```diff
@@ -1,34 +1,42 @@
...
 
-    var options = {
+    let options = {
         'method': 'POST',
         'url': config.lcdURL + '/bank/accounts/' + address + '/transfers',
         'headers': {
             'Content-Type': 'application/json'
         },
-        body: JSON.stringify({"base_req":{"from":config.testAccountAddress,"chain_id":config.chain_id},"amount":[{"denom":"stake","amount":"1000000"}]})
+        body: JSON.stringify({
```
Comment (deepanshutr) [inline transaction/bank/sendCoin.js:16]: remove hardcoding

Comment (aditim121) [reply transaction/bank/sendCoin.js:16]: Resolved

File: transaction/accounts/create.js (hunk 3 above)
Comment (aditim121) [reply transaction/accounts/create.js]: Resolved

File: cosmos160phc9aajqmllhn442tyehkrwpd8sytr6q9uwc.json (hunk 2 above)
Comment (aditim121) [reply cosmos160phc9aajqmllhn442tyehkrwpd8sytr6q9uwc.json]: Resolved

File: cosmos1pkkayn066msg6kn33wnl5srhdt3tnu2vzasz9c.json (hunk 1 above)
Comment (aditim121) [reply cosmos1pkkayn066msg6kn33wnl5srhdt3tnu2vzasz9c.json]: Resolved

**Review by deepanshutr — APPROVED** [review]
Reviewed, all issues resolved.

**Pull request merged**